mongo/client_examples_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package mongo_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+func ExampleConnect_ping() {
```

Comment:
I'm not clear how these examples look when rendered in GoDoc.  Are they all stacked on top of each other?  What does the heading look like?

Reply:
They're stacked in alphabetical order. The heading looks like `Example (Ping)`.